.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+	"recommendations": [
+		"ms-vscode.cpptools",
+		"ms-vscode.cpptools-extension-pack",
+    "ms-vscode.cmake-tools",
```

Comment:
is there a reason this is indented differently?

Reply:
ah thx - tab vs spaces... switched to spaces.